### PR TITLE
Update targets

### PR DESCRIPTION
**Pull request by martinvol**

### CODE REVIEW

File: CGPs/cgp-0132/alfajores.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        "0",
+        "0"
+      ],
+      "value": "0"
+    },
+    {
+        "contract": "EpochRewards",
+        "function": "setTargetVotingYield",
+        "args": [
+            "235000000000000000000"
```
Comment (rcroessmann) [inline CGPs/cgp-0132/alfajores.json:16]:
```suggestion
            "101500000000000000000"
```

File: CGPs/cgp-0132/alfajores.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        ],
+        "value": "0"
+    },
+    {
+        "contract": "EpochRewards",
+        "function": "setTargetVotingYieldParameters",
+        "args": [
+            "500000000000000000000",
+            "1130000000000000000"
```
Comment (rcroessmann) [inline CGPs/cgp-0132/alfajores.json:25]:
```suggestion
            "0"
```

File: CGPs/cgp-0132/alfajores.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        ],
+        "value": "0"
+    },
+    {
+        "contract": "EpochRewards",
+        "function": "communityRewardFraction",
+        "args": [
+            "250000000000000000000000"
```
Comment (rcroessmann) [inline CGPs/cgp-0132/alfajores.json:33]:
```suggestion
            "10000000000000000000000"
```

**Comment by ericnakagawa** [comment]
Hi @rcroessmann @martinvol, it looks like you folks are editing cgp-0132. cgp-0132 is now the stabila proposal. You folks should look into using cgp-0133 or cgp-0134.

File: CGPs/cgp-0132/alfajores.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+        "function": "setTargetVotingYieldParameters",
+        "args": [
+            "500000000000000000000",
+            "0"
+        ],
+        "value": "0"
+    },
+    {
+        "contract": "EpochRewards",
+        "function": "communityRewardFraction",
```
Comment (martinvol) [inline CGPs/cgp-0132/alfajores.json:31]:
```suggestion
        "function": "setCommunityRewardFraction",
```

**Comment by 0xzoz** [comment]
@martinvol could we please get your insight on the parameter changes that need to be added to the JSON in #436. I was asked to help submit the proposal for the proposers and there are some comments that need to be addressed as I was using this as a reference. Thanks

**Comment by 0xzoz** [comment]
Closed due to #436 being submitted

**Pull request closed without merging**